app/services/combined_audio_derivative_creator.rb

```diff
@@ -0,0 +1,170 @@
+require 'tempfile'
+
+
+Response = Struct.new(:webm_file, :mp3_file, :fingerprint, :start_times, keyword_init: true)
+#
+# Generate the combined audio derivatives for a
+# given oral history work.
+#
+# some_work = Work.find_by_friendlier_id(friendlier_id)
+# CombinedAudioDerivativeCreator.new(some_work).generate
+#
+# Sample output:
+# <struct Response
+#  webm_file=<File:/path/to/file.webm>,
+#  mp3_file=<File:/path/to/file.mp3>,
+#  fingerprint="e0248c40015d6dc90b0d02937950b5d7",
+#  start_times=
+#   [["ddbd1a8d-c2eb-47b3-85a4-11b4ffb41719", 0],
+#    ["df773502-56d7-4756-a58c-b1f479910e97", 1.593469]]
+#  >
+
+class CombinedAudioDerivativeCreator
+
+  attr_reader :work
+
+  def initialize(work)
+    @cmd = TTY::Command.new(printer: :null)
+    @work = work
+  end
+
+  def generate
+    output_files = {}
+
+    # Create the two output files:
+    ['mp3', 'webm'].each do |format|
+      output_files[format] = output_file(format)
+      ffmpeg_args = args_for_ffmpeg(output_files[format].path)
+      log "Creating #{format} using: #{ffmpeg_args.join(" ")}"
+      @cmd.run(*ffmpeg_args, binmode: true)
```

Comment:
Does TTY::Command not already log itself?  If it doesn't it might be becuase we TOLD it not to! `TTY::Command.new(printer: :null)`? Instead of telling TTY::Command not to log, and then logging separately, can we just tell it to log instead? Only if there's a way to tell it to log at our desired level I guess?

Alternately, we could just not log it in case of successful completion -- the places you copied where I was doing `printer: null` are places whereI decided I didn't want it clogging up the logs (although in those places it was much more frequent things).

Reply:
Turned on logging for cmd, removed all logs from the code!